src/pages/docs/channels/options/encryption.mdx

```diff
@@ -30,6 +30,24 @@ Other attributes of messages and presence messages, such as event `name` and `cl
 
 The key that's in use at any given time is known by the SDK. The Ably service only knows that a given message payload was encrypted, not the key used to encrypt it. When accessing messages using the history feature, it is the caller's responsibility to ensure that the correct key is configured for the channel before the history request is made.
 
+<If lang="javascript">
```

Comment:
Ideally this would should be React, and we'd provide some relevant sample code in the blocks.